hbase-mapreduce/src/main/java/org/apache/hadoop/hbase/mapreduce/HFileOutputFormat2.java

```diff
@@ -222,6 +222,7 @@ public RegionLocator getRegionLocator() {
       private final Map<byte[], WriterLength> writers = new TreeMap<>(Bytes.BYTES_COMPARATOR);
       private final Map<byte[], byte[]> previousRows = new TreeMap<>(Bytes.BYTES_COMPARATOR);
       private final long now = EnvironmentEdgeManager.currentTime();
+      private byte[] tableNameBytes = Bytes.toBytes(writeTableNames);
```

Comment:
Should we do this under writeMultipleTables check?
private byte[] tableNameBytes = (writeMultipleTables)? null: Bytes.toBytes(writeTableNames);

Reply:
yes，i want to make code simple, so just keep init no matter multiple or not 

It is a good ponit to  use use writeMultipleTables  check,will address it